include/Utility.hpp

```diff
@@ -16,53 +16,56 @@ int sampleIndex(std::string sample_name);
 
 template <typename T> std::vector<T> deepCopy(const std::vector<T> &vec)
 {
-	// The purpose of this function is to create a deep copy of a vector.
-	std::vector<T> copy;
-	for (T const &elem : vec) {
-		copy.push_back(elem);
-	}
-	return copy;
+    // The purpose of this function is to create a deep copy of a vector.
+    std::vector<T> copy;
+    for (T const &elem : vec) {
+        copy.push_back(elem);
+    }
+    return copy;
```

Comment:
I think vector copy is already deep by default??? Not a huge deal especially since I don't know specifics of where this is used

Reply:
We have it here to 100% guarantee deep copy works as intended. It is also more explicit and helps us match legacy code.